libethereum/Executive.cpp

```diff
@@ -411,7 +427,11 @@ bool Executive::executeCreate(Address const& _sender, u256 const& _endowment, u2
     // Schedule _init execution if not empty.
     if (!_init.empty())
         m_ext = make_shared<ExtVM>(m_s, m_envInfo, m_sealEngine, m_newAddress, _sender, _origin,
-            _endowment, _gasPrice, bytesConstRef(), _init, sha3(_init), m_depth, true, false);
+            _endowment, _gasPrice, bytesConstRef(), _init, sha3(_init), _version, m_depth, true,
+            false);
+    else
+        // code stays empty, but we set the version
+        m_s.setCode(m_newAddress, {}, _version);
```

Comment:
What's the point of setting the version if there's no code?

Reply:
This question might be delegated to the EIP.

Reply:
I discussed this with EIP author here https://github.com/sorpaas/EIPs/issues/2#issuecomment-506363868